EpsilonCCS/Inc/Data/DriverControlData.h

```diff
@@ -57,14 +46,26 @@ struct MusicInputs
     unsigned char volumeDown;
     unsigned char nextSong;
     unsigned char prevSong;
-}
+};
 
 struct DriverInputs
 {
-    unsigned char brakes
-    unsigned char forward
-    unsigned char reverse
-    unsigned char pushToTalk
-    unsigned char horn
-    unsigned char reset
-}
+    unsigned char brakes;
+    unsigned char forward;
+    unsigned char reverse;
+    unsigned char pushToTalk;
+    unsigned char horn;
+    unsigned char reset;
+};
+
+struct DriverControlData
+{
+    unsigned char driverControlsBoardAlive;
+    struct LightsInputs lightsInputs;
+    struct MusicInputs musicInputs;
+    unsigned short int acceleration;
+    unsigned short int regenBraking;
+    struct DriverInputs driverInputs;
+};
+
+extern struct DriverControlData driverControlData;
```

Comment:
This line doesn't have a new line after it, I'm not sure if that matters much, but the other files do.